### DISCUSSION TITLE
Appel de API externe à FRW

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#140] [Annonces]
Bonjour,

Nous avons des formulaires d'où le bouton final "Soumettre" doit appeler un API dans notre système. Lorsque nous testons un de nos formulaire (dans votre environnement IT), nous devons pointer vers le bon API dépendamment de l'environnement  (satellite/unitaire/acceptation). Comme le formulaire est commun aux trois environnements, comment fait-on pour appelé le bon API de l'environnement ?

Merci,

**Comment by loima02** [comment]
Bonjour,

Deux options s'offrent à vous

Option 1) Un système autorisé par pallier
- Vous avez déjà un système autorisé en QA (exemple : 700-000). Il est possible pour vous d'obtenir un système autorisé pour un autre pallier de votre infrastructure. Par exemple, 700-001 pour satellite, 700-002 pour unitaire, etc.
- À ce moment, chaque système autorisé aura son fichier de configuration et de transmission et pourra être ajusté pour pointer sur le pallier de votre choix

Option 2) Un formulaire par pallier
- Le fichier de transmission peut être personnalisé dans chaque formulaire. Il est possible pour vous de créer un formulaire par pallier. Par exemple, 3003-satellite, 3003-unitaire, etc.
- À ce moment, chaque formulaire aura son fichier de transmission et pourra être ajusté pour pointer sur le pallier de votre choix

Si vous avez des questions, n'hésitez pas!

Merci et bonne journée!

**Comment by Hutson48** [comment]
Sera corrigé avec #229
